data/sheragi outfits.txt

```diff
@@ -0,0 +1,213 @@
+	"shields" -2000
+	"hull" -6000
+	"energy consumption" 3
+	"heat generation" 4
+	"jump drive" -1
+	unplunderable 1
+	description "There is a lot of broken Sheragi hardware in here. You need to fix that before you can take the ship back into Human space."
+
+
+# Weapons
+
+outfit "Dragonflame Cannon"
+	category "Secondary Weapons"
+	cost 150000000
+	thumbnail "outfit/dragonflame"
+	"mass" 156
+	"outfit space" -156
+	"weapon capacity" -156
+	"required crew" 10
+	"gun ports" -1
+	weapon
+		sound "dragonflame" 
+		icon "icon/dragonflame"
+		"fire effect" "dragonflame"
+		"hit effect" "bright hit"
+		"hit effect" "dragonflame plasma" 30
+		"hit effect" "dragonflame debris" 50
+		"velocity" 2000
+		"lifetime" 1
+		"reload" 180
+		cluster
+		"firing energy" 16000
+		"firing heat" 40000
+		"firing force" 1000
+		"shield damage" 17000
+		"firing fuel" 300
+		"hull damage" 9000
+		"piercing" 0.3
+		"heat damage" 40000
+		"hit force" 2000
+	description "This ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is prohibitive, and it's hard to imagine what could motivate the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, and directs most of the energy of the explosion forwards in a devastatingly powerful beam of gamma radiation."
+
+outfit "Sheragi Anti-Missile"
+	category "Turrets"
+	cost 845000
+	thumbnail "outfit/sheragiam"
+	"mass" 35
+	"outfit space" -35
+	"weapon capacity" -35
+	"turret mounts" -1
+	weapon
+		"hardpoint sprite" "hardpoint/sheragiam"
+		"hardpoint offset" 1.5
+		"hit effect" "sheragiam"
+		"anti-missile" 200
+		"velocity" 450
+		"lifetime" 1
+		"reload" 60
+		"firing energy" 220
+		"firing heat" 440
+	description "The Sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but takes a long time to recharge. It was probably made to counter nuclear missiles."
+
+outfit "Particle Waveform Turret"
+	category "Turrets"
+	cost 1650000
+	thumbnail "outfit/pwave turret"
+	"mass" 47
+	"outfit space" -47
+	"weapon capacity" -47
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/pwavecannon"
+			"frame rate" 20
+		sound "pwave"
+		"hardpoint sprite" "hardpoint/pwavehp"
+		"hardpoint offset" 11
+		"hit effect" "pwave impact"
+		"inaccuracy" 1.5
+		"velocity" 50
+		"lifetime" 18
+		"reload" 60
+		"firing energy" 180
+		"firing heat" 651
+		"shield damage" 110
+		"hull damage" 270
+		"heat damage" 1344
+		"turret turn" 1.8
+	description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and easily melts through matter. By the nature of these weapons, one can deduce their tactic was to hit with the Dragonflame Cannon at long range to break the enemy shields and finish them off in relatively closer range with the Particle Waveform Cannon."
```

Comment:
description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and easily melts through matter. By the nature of these weapons, one can deduce that the tactic used was to use the Dragonflame Cannon at long range to break the enemy shields and finish the ship off at relatively close range with the Particle Waveform Cannon."